src/pages/homePage/notificationPage.js

```diff
@@ -0,0 +1,14 @@
+import React, { useState } from 'react';
+import { SafeAreaView } from 'react-native';
+import EventStyles from './eventStyles';
+import TopBar from '../../components/TobBar.js';
+
+const NotificationPage = () => {
+    return (
+        <SafeAreaView style={[EventStyles.container, { alignItems: 'center' }]}>
+            <TopBar tobBar="알림"/>
```

Comment:
```suggestion
<TopBar topBar="알림"/>
```
오타가 있네요 :) TobBar.js 파일명 오타도 수정 부탁드립니다!